src/test/java/com/wikia/webdriver/common/testnglisteners/InvokeMethodAdapter.java

```diff
@@ -17,11 +17,14 @@
   public void afterInvocation(IInvokedMethod method, ITestResult result) {
     if (method.isTestMethod()) {
       List verificationFailures = Assertion.getVerificationFailures(result);
-      // if there are verification failures...
+      // if there are verification failures... jesli assercja sie wywalila jakakolwiek
       if (PageObjectLogging.getVerificationStack().contains(false)) {
         result.setStatus(ITestResult.FAILURE);
         result.setThrowable(new TestFailedException(result.getThrowable()));
+//        Throwable cause = result.getThrowable()==null ? PageObjectLogging.lastThrow : result.getThrowable();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)